op-acceptance-tests/tests/interop/seqwindow/expiry_test.go

```diff
@@ -110,6 +131,38 @@ func TestSequencingWindowExpiry(gt *testing.T) {
 	other := sys.L2ELA.BlockRefByNumber(safe.Derived.Number)
 	require.Equal(safe.Derived.Hash, other.Hash, "supervisor must match chain with EL")
 
+	t.Logger().Info("Re-enabling batch-submitter")
 	// re-enable the batcher now that we are done with the test.
 	t.Require().NoError(sys.L2BatcherA.ActivityAPI().StartBatcher(t.Ctx()))
+	// TODO: batcher submits future span batch, misses a L2 block
```

Comment:
when should this TODO be addressed?

Reply:
tracked here: https://github.com/ethereum-optimism/optimism/issues/16036